<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SigCaceresDialogBase</class>
 <widget class="QDialog" name="SigCaceresDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIG Cáceres</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Servicios</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:0.5px solid rgb(229, 229, 229);
border-bottom-color: rgb(188, 188, 188)


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources.qrc">:/plugins/sig_caceres/iconos/logo-sig.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_icon_ayti">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources.qrc">:/plugins/sig_caceres/iconos/logo-ayto.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_centrosadministrativos">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Centos administrativos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_carga_centrosadministrativos">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>cargar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Centros sanitarios</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_centrossanitarios">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Servicios turísticos</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_serviciosturisticos">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ocio y entretenimiento</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_ocioentretenimiento">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mobiliario urbano</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_mobiliariourbano">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_carga_centrossanitarios">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>cargar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_carga_serviciosturisticos">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>cargar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_carga_ocioentretenimiento">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>cargar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_carga_mobiliariourbano">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>cargar</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>label_icon</zorder>
  <zorder>label_icon_ayti</zorder>
  <zorder>combo_urbanavectorial</zorder>
  <zorder>label_2</zorder>
  <zorder>b_carga_urbanavectorial</zorder>
  <zorder>label_4</zorder>
  <zorder>combo_urbanaescaneada</zorder>
  <zorder>label_5</zorder>
  <zorder>combo_tmvectorial</zorder>
  <zorder>label_6</zorder>
  <zorder>combo_tmescaneada</zorder>
  <zorder>label_7</zorder>
  <zorder>combo_cartografiaactualizada</zorder>
  <zorder>b_carga_urbanaescaneada</zorder>
  <zorder>b_carga_tmvectorial</zorder>
  <zorder>b_carga_tmescaneada</zorder>
  <zorder>b_carga_cartografiazonas</zorder>
 </widget>
 
 <connections/>
</ui>
